pallets/parachain-system/src/lib.rs

```diff
@@ -1125,13 +1125,39 @@ pub trait OnSystemEvent {
 	/// Called when the validation code is being applied, aka from the next block on this is the new runtime.
 	fn on_validation_code_applied();
 }
+/// Holds the relay chain state root and block number.
```

Comment:
docs: worth mentioning that this is only the metadata about the most recent relay-parent of this fork of the parachain